schemas/service.proto

```diff
@@ -315,6 +315,7 @@ message TaskWorkerHeartBeatRequest {
 
 message RegisterTaskWorkerResponse {
   repeated LHHostInfo your_hosts = 1;
+  optional bool is_cluster_healthy = 2;
```

Comment:
It says if the cluster is able to process hearbeat for the task worker. Implementation of this will be more complex in the future.

Reply:
1. After talking to Edu offline, it seems that the `optional` is so that _new clients_ can talk to _old servers._ Makes sense